### ISSUE TITLE
Custom base colors not drawn on cell selection

### ISSUE THREAD

**Issue by gradha** [akiroom/AXRatingView#23]
Using the following modification to the current sample:
```diff
diff --git a/AXRatingViewDemo/AXRatingViewDemo/AXOnTheCellViewController.m b/AXRatingViewDemo/AXRatingViewDemo/AXOnTheCellViewController.m
index d5f33bb..685a3c8 100644
--- a/AXRatingViewDemo/AXRatingViewDemo/AXOnTheCellViewController.m
+++ b/AXRatingViewDemo/AXRatingViewDemo/AXOnTheCellViewController.m
@@ -55,12 +55,14 @@
     [ratingView addTarget:self action:@selector(changeRate:) forControlEvents:UIControlEventValueChanged];
     cell.accessoryView = ratingView;
     [cell.accessoryView sizeToFit];
+    ratingView.baseColor = [UIColor colorWithRed:1 green:0 blue:0 alpha:1];
+	//ratingView.baseColor = [UIColor greenColor];
   }
   [cell.textLabel setText:[NSString stringWithFormat:@"No.%d", (int)(indexPath.row + 1)]];
   
   // Setup rating view for each cell.
   AXRatingView *ratingView = (AXRatingView *)cell.accessoryView;
-  ratingView.value = [_ratingNumbers[indexPath.row] floatValue];
+  ratingView.value = 3;
   ratingView.tag = indexPath.row;
   
   return cell;
```
…will change the base color to red and force all cells to three stars. When a cell is highlighted during touch, the red stars won't be drawn as shown in this screenshot:

![red base](https://cloud.githubusercontent.com/assets/101927/24760114/f47ec90a-1ae7-11e7-851a-ac1a686fd592.png)

However, if the color set to `baseColor` is one of the defaults provided by the `UIColor` class, then it *will* show during highlight. Why? How can this be avoided with a custom color?

![green base](https://cloud.githubusercontent.com/assets/101927/24760141/146c1f24-1ae8-11e7-87c3-81fb1406c9bb.png)

**Comment by akiroom** [comment]
Sorry for late. It would be caused by the spec of UITableViewCell that it changes the background of child views when it is selected.

Maybe I can fix it. I will try.

**Comment by akiroom** [comment]
I fixed it in #27. I will release soon.
